esp32/README.md

```diff
@@ -91,6 +91,11 @@ $ git submodule init lib/berkeley-db-1.xx
 $ git submodule update
 ```
 
+You will also need to have PySerial installed if it isn't already.
+```bash
+$ pip install pyserial
+```
+
```

Comment:
I've put this in based on where I've seen the error. I could get up to this point without having pyserial installed, but then `make` for ESP32 threw an error. I can move it up to the ESP IDF if you prefer.

The ESP IDF docs do mention pyserial (http://esp-idf.readthedocs.io/en/latest/get-started/linux-setup.html?highlight=pyserial) but apt-get will just install for the system python (2.7 on my debian machine). If you're installing with a non-default python it doesn't help.